### DISCUSSION TITLE
Recieving shadows are pitch black

### DISCUSSION THREAD

**Discussion by Saverix339** [bnpr/Malt#449] [Q&A]
![uno](https://user-images.githubusercontent.com/98220222/202769944-c9e0eeda-82dc-4764-a6a7-4fa463fbd312.JPG)
This is the normal material for the eye, with no shadows.
![duee](https://user-images.githubusercontent.com/98220222/202770058-eac829ef-8a0d-46c3-b83e-6dc4b2e3e262.JPG)
This is the second material, with shadows on. As you can see, the shadows that the face casts on the eyes are pitch black. Is there amy way to make them smoother?

**Comment by pragma37** [comment]
Shadow color is whatever you set in the Base Color.
